internal/pkg/template/templates/environment/cf.yml

```diff
@@ -585,14 +585,20 @@ Resources:
 {{include "ar-vpc-connector" . | indent 2}}
 {{- end}}
 {{- if .VPCConfig.FlowLogs }}
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Metadata:
+      'aws:copilot:description': 'A  log group to create a flow log for the VPC'
+    Properties:
+      RetentionInDays: 14
```

Comment:
I get that we are setting its default value to 14. But how will customers set their specific value of RetentionInDays?
Or have we just decided to set the default value to 14 and not let customers modify it?

Reply:
+1 for customizable log retention period.
One use-case I can think of is log retention requirements due to compliance reasons.